rules.yaml

```diff
@@ -97,9 +99,12 @@
 - { name: xpi-adblock_plus,            setname: adblock-plus,             repos: [ freebsd ] }
 - { name: xpi-foxyproxy,               setname: foxyproxy,                repos: [ freebsd ] }
 - { name: zookeeper-c,                 setname: zookeeper,                repos: [ gentoo ]  }
+- { namepat: "android-platform-system-(core|extras)", setname: android-tools, repos: [ debian ] }
```

Comment:
Mapping Android stuff is going to be a mess either way:
- fastboot (like adb) lives in `core` repo but has dependencies in `extras`, `libselinux`.
- simpleperf lives in `extras` but has dependencies in `core`, `bionic` and `lzma` (in trunk).
- Fedora (and RPM-friends), Slackware, Gentoo, Arch have adb/fastboot in android-tools
- Arch and Debian also have adb/fastboot in *-platform-tools with Android SDK versioning scheme